ggshield/cmd/iac/scan/diff.py

```diff
@@ -111,16 +113,34 @@ def iac_scan_diff(
 ) -> Optional[IaCDiffScanResult]:
     config = ctx.obj["config"]
     client = ctx.obj["client"]
-
+    verbose = config.user_config.verbose if config and config.user_config else False
+    if verbose:
+        click.echo(f"> Scanned files in reference {ref}", err=True)
```

Comment:
You can use `text_utils.display_info()` for all these instead of `click.echo(, err=True)`